tfx/examples/imdb/imdb_dataset_utils.py

```diff
@@ -0,0 +1,41 @@
+# Lint as: python2, python3
+# Copyright 2019 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
```

Comment:
Let's keep this in the README.md file above. Embedding it in the util file could result it being lost when people refactor this python source code.

Also wrap your python snippets with:

\`\`\`python
some code
\`\`\`

Reply:
fixed